README.md

```diff
@@ -1,12 +1,22 @@
 # Pulumi CDK Adapter (preview)
 
-The Pulumi CDK Adapter is a library that enables [Pulumi](https://github.com/pulumi/pulumi) programs to use [AWS CDK](https://github.com/aws/aws-cdk) constructs.
+The Pulumi CDK Adapter is a library that enables
```

Comment:
Is this change necessary? Looks like Emacs fil-paragraph got applied. I think we prefer to have the previous form.

Reply:
I don't know, I kind of prefer the 80 line length limit. It makes looking/editing so much easier. It's also the default in most markdown linters, (e.g. [markdownlint](https://github.com/markdownlint/markdownlint/blob/main/docs/RULES.md#md013---line-length)

Reply:
AH nice, one of those things that'd just nice to have a standard. We can adopt it in the project.  I'll setup column width 80 in Emacs.